### DISCUSSION TITLE
Skipping index operations in `init_beanie(...)`

### DISCUSSION THREAD

**Discussion by gencurrent** [BeanieODM/beanie#924] [Feature Request]
Looks like it's a complete duplicate of https://github.com/roman-right/beanie/discussions/608.
I would like to reraise this feature or directly create a PR or ask for any update on this.

Nevertheless.
The text I wanted to write before I found the discussion is:
---------
I have 2 services where
* The first one initializes the DB;
* The second utilizes the DB structure without any DB [re]initialization.
Thus I think it's reasonable to add a keyword argument to the `init_beanie` function to skip either DB initialization or indexes operations.
For example, it might be just `skip_indexes`.

The current workaround I started using is:  
```
class IndexlessBeaineInitializer(Initializer):
    """
    Beanie initializer subclass to skip indexes operations
    """

    async def init_indexes(self, cls, allow_index_dropping: bool = False):
        pass
```
which does the job well. 

---------

**Comment by todowa** [comment]
Yes agreed @gencurrent . I have a read-only analytics service, and `init_beanie` doesn't work with read-only users because it's unable to create indexes.
